maven_task4/src/test/java/org/Marta/AgeVerificationTest.java

```diff
@@ -0,0 +1,21 @@
+package org.Marta;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+class AgeVerificationTest {
+
+    @Test
+    void isLegalAgeTest() {
+        int age1 = 1;
+        int age2 = 17;
+        int age3 = 18;
+        int age4 = 32;
+        int age5 = 100;
+
```

Comment:
to powinny być osobne testy. Jeden test - jedna asercja. ;)

Reply:
zrobione ;)